tests/acceptance/features/bootstrap/GraphContext.php

```diff
@@ -2317,4 +2317,66 @@ public function userGetsThePersonalDriveInformationOfUserUsingGraphApi(string $b
 			)
 		);
 	}
+
+	/**
+	 * @When /^the user "([^"]*)" exports (?:her|his) GDPR report to "([^"]*)" using the Graph API$/
+	 *
+	 * @param string $user
+	 * @param string $path
+	 *
+	 * @return void
+	 * @throws GuzzleException
+	 */
+	public function userGeneratesGDPRReportOfOwnDataToPath(string $user, string $path): void {
+		$credentials = $this->getAdminOrUserCredentials($user);
+		$userId = $this->featureContext->getAttributeOfCreatedUser($user, 'id');
+		$this->featureContext->setResponse(
+			GraphHelper::generateGDPRReport(
+				$this->featureContext->getBaseUrl(),
+				$this->featureContext->getStepLineRef(),
+				$credentials['username'],
+				$credentials['password'],
+				$userId,
+				$path
+			)
+		);
+		$this->featureContext->pushToLastStatusCodesArrays();
+	}
+
+	/**
+	 * @Then the downloaded JSON content should contain event type :key in item 'events' and should match
```

Comment:
Would this make sense in this case?
```suggestion
	 * @Then the downloaded JSON content of event type : key in item 'events'  should match
```

Reply:
we have `And the JSON data of the response should contain the group "tea-lover" in the item 'value', the group-details should match` such steps similar to it. May be similar like this would be nice.